AppCenterReactNativeShared/ios/AppCenterReactNativeShared/AppCenterReactNativeShared.m

```diff
@@ -64,4 +68,8 @@ + (void) setWrapperSdk:(MSWrapperSdk *)sdk {
     [MSAppCenter setWrapperSdk:sdk];
 }
 
++ (void) setStartAutomatically:(BOOL)startedAutomatically {
```

Comment:
nit - What is the reason to name it `(BOOL)startedAutomatically`? I prefer `(BOOL)shouldStartAutomatically` or `(BOOL)startAutomatically`.

Reply:
name conflict with the static variable

Reply:
On Android we have the getter `isStartedAutomatically`, removing the `ed` is odd.
should: then we would have a setter with `setShouldStartAutomatically` which would be odd.

On Android is just the couple is/setStartedAutomatically like every boolean "property".

Reply:
Just adding this for completeness:

If we define the static property like this

`@property (class, nonatomic, assign) BOOL isStartedAutomatically;`, provide an ivar `static BOOL _isStartedAutomatically = NO`;
and we define getter and setter like this

```
+ (BOOL)isStartedAutomatically {
  return _isStartedAutomatically;
}

+ (void)setIdentifier:(BOOL)isStartedAutomatically {
  _isStartedAutomatically = isStartedAutomatically;
}

``` there should not be any naming conflicts.